src/client/src/components/FileButton/__test__/index.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { FileButton } from '..';
+
+import { Provider } from 'mobx-react';
+import * as appModels from '@app/models';
+import { ExtensionType } from '@app/types';
+
+test('Should render properly', () => {
+    const props = {
+        handleClick: () => {
+            console.log('Clicked');
+        },
+        extension: ExtensionType.JS,
+        name: 'main',
+    };
+
+    const { container } = render(
+        <Provider {...appModels}>
+            <FileButton {...props} />
+        </Provider>,
+    );
+
+    expect(container.firstChild).toHaveClass('file-button');
+
+    // fireEvent.click(container.firstChild);
+    // expect(setActiveIndex).toBeCalledTimes(1);
+
+    // fireEvent.click(container.lastChild);
+    // expect(setActiveIndex).toBeCalledTimes(2);
+});
```

Comment:
It is the base test that I've written for testing how does it work in a custom webpack config. Seems to work well. Moving forward to do some fancy stuff :)

Reply:
Done. The editor, for now, has a test case where the component is fully covered with the test. Moving forward to cover all components with tests and coverage with our rules.